tests/integration/solidity_contracts/conftest.py

```diff
@@ -140,18 +145,22 @@ async def _factory(contract_app, contract_name, *args, **kwargs):
 
 
 @pytest.fixture(scope="package")
-def create_account_with_bytecode(starknet, kakarot, deploy_bytecode, deploy_eoa):
+def create_account_with_bytecode_and_storage(
+    starknet, kakarot, deploy_bytecode, deploy_eoa
+):
     """
     Fixture to create a solidity contract in kakarot without running the bytecode.
-    The given bytecode is directly stored into the account, similarly to what is done
-    in a genesis config.
+    The given bytecode and storage are directly stored into the account, similarly to
+    what is done in a genesis config.
 
     Returns the corresponding starknet contract with the extra evm_contract_address attribute.
     """
 
-    async def _factory(bytecode: str, caller_eoa=None):
+    async def _factory(
+        bytecode: str = "", storage: Dict[str, str] = {}, caller_eoa=None
```

Comment:
never put mutable argument as default value in python, because the updates will be kept for forthcoming call to the function

Reply:
Gotcha that's a good point, thanks I've updated it